iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -21,11 +21,20 @@
 #include "iceoryx_posh/internal/runtime/posh_runtime_impl.hpp"
 
 #include <cstdint>
+#include <new>
+#include <type_traits>
 
 namespace iox
 {
 namespace runtime
 {
+
+// A refcount for use in getLifetimeParticipant().
+static int refcount;
+// Track whether the refcount lifetime mechanism is used by this particular
+// implementation of the abstract PoshRuntime class.
+static bool manual_lifetime_management = false;
```

Comment:
We use the universal initialization approach and lower camel case for variables. Please write
```
static bool manualLifetimeManagement{false};
```

Reply:
This must also be an atomic.

Reply:
Should it also have a `s_` prefix? I assume it stands for "static", right?

Reply:
Yes. Just to have an indicator that it is not a local variable.